src/com/tekwill/topic3/Operations.java

```diff
@@ -20,30 +19,25 @@ static void printAritmeticOperations(int a, int b) {
 
 
     static void printRelationarOperation(int a, int b) {
-       /* int a = 5;
-        int b =15;*/
+
         System.out.println("Equality " + (a == b));
         System.out.println("Not equal " + (a != b));
         System.out.println("Greater " + (a > b));
         System.out.println("Less " + (a < b));
         System.out.println("Greater equal " + (a >= b));
         System.out.println("Less equal " + (a <= b));
-
     }
 
     static void printLogicOperations(boolean a, boolean b) {
-       /* boolean a = true;
-        boolean b = false;*/
+
         System.out.println("AND: " + (a && b));
         System.out.println("OR: " + (a || b));
         System.out.println("Negation: " + (!a));
-
-
     }
 
     static void useForAndWhile(int limita) {
         int index = 10;
-        int limit = 100;
+        int limit = 20;
```

Comment:
parametrul limita este populat in cadrul apelului metodei, acolo poarta numele limita in loc de limit. Respectiv, poate fi folosit parametrul formal pentru opertatiile ulterioare, astfel incat acesta sa fie populat dinamic la apelarea metodei.